catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/pom.xml

```diff
@@ -269,7 +269,7 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.80</minimum>
+                                            <minimum>0.78</minimum>
```

Comment:
Note that test coverage did not actually decrease with this change. The instruction coverage was updated to reflect its true value; LenientLimit has prevented this from failing the build.